src/main/java/com/gluonhq/substrate/util/windows/MSIBundler.java

```diff
@@ -0,0 +1,325 @@
+            Logger.logSevere("Please make sure to install Wix Toolset (https://wixtoolset.org/) before proceeding.");
+            return false;
+        }
+
+        final String appName = projectConfiguration.getAppName();
+        final String version = Optional.ofNullable(projectConfiguration.getReleaseConfiguration().getVersionName()).orElse(DEFAULT_APP_VERSION);
+        Path localAppPath = paths.getAppPath().resolve(appName + ".exe");
+        if (!Files.exists(localAppPath)) {
+            throw new IOException("Error: " + appName + ".exe not found");
+        }
+        Logger.logInfo("Building exe for " + localAppPath);
+
+        /**
+         * Create directories and copy resources
+         */
+        Path tmpMSI = paths.getTmpPath().resolve("tmpMSI");
+        if (Files.exists(tmpMSI)) {
+            FileOps.deleteDirectory(tmpMSI);
+        }
+        Files.createDirectories(tmpMSI);
+
+        Path userAssets = rootPath.resolve(Constants.WIN_ASSETS_FOLDER);
+        // copy assets to gensrc/windows
+        Path windowsGenSrcPath = paths.getGenPath().resolve(sourceOS);
+        Path windowsAssetPath = windowsGenSrcPath.resolve(Constants.WIN_ASSETS_FOLDER);
+        Files.createDirectories(windowsAssetPath);
+        // Copy system resources and over-write it with user provided resources
+        FileOps.copyDirectoryFromResources("/native/windows/assets", windowsAssetPath);
+        Logger.logInfo("Default icon.ico image generated in " + windowsAssetPath + ".\n" +
+                "Consider copying it to " + rootPath + " before performing any modification");
+        if (Files.exists(userAssets)) {
+            FileOps.copyDirectory(userAssets, windowsAssetPath);
+        }
+
+        Path config = tmpMSI.resolve("config");
+        Files.createDirectories(config);
+        Path wixPath = config.resolve("main.wxs");
+        Path wixObjPath = wixPath.getParent().resolve(wixPath.getFileName() + ".wixobj");
+        Path msiPath = paths.getGvmPath().getParent().resolve(appName + "-" +  version  + ".msi");
+        FileOps.copyResource("/native/windows/wix/main.wxs", wixPath);
+        FileOps.copyDirectory(windowsAssetPath, config);
+
+        /**
+         * Wix Compile
+         */
+        Map<String, String> userInput = createAppDetailMap();
+        List<String> processArgs = new ArrayList<>(List.of(
+                WixTool.CANDLE.getPath(),
+                "-nologo",
+                wixPath.toString(),
+                "-ext", "WixUtilExtension",
+                "-arch", "x64",
+                "-out", wixObjPath.toString()
+        ));
+
+        userInput.entrySet().stream()
+                .map(wixVar -> String.format("-d%s=%s", wixVar.getKey(), wixVar.getValue()))
+                .forEachOrdered(processArgs::add);
+        Logger.logInfo(String.join(" ", processArgs));
+        ProcessRunner candle = new ProcessRunner(processArgs.toArray(new String[0]));
```

Comment:
With the changes I've mentioned before, I get a valid command line, but the process still fails:
```
Output 
====== 
candle.exe : error CNDL0103 : The system cannot find the file 'Gluon' with type 'Source'.   
Result
 ====== 
result: 103
```
However if I copy that command line from the log and paste it directly on a terminal it works fine.

Reply:
Do you have a reproducible sample for this?

Reply:
I’m using a project generated from start.gluon.io, but any project with `<name>My Application</name>` in its pom will do.